frontend/public/kubevirt/components/cluster/cluster-overview.jsx

```diff
@@ -134,7 +134,7 @@ export class ClusterOverview extends React.Component {
     };
     this.fetchAndStore(`${k8sBasePath}/healthz`, 'k8sHealth', handleK8sHealthResponse, coFetch);
     this.fetchAndStore(
-      `${k8sBasePath}/apis/subresources.${VirtualMachineModel.apiGroup}/${VirtualMachineModel.apiVersion}/healthz`,
+      `/apis/subresources.${VirtualMachineModel.apiGroup}/${VirtualMachineModel.apiVersion}/healthz`,
```

Comment:
LGTM - just wondering how did this work before? was there a recent change in Kubevirt?

Reply:
Not that I am aware of maybe @davidvossel @rmohr ?

Reply:
Hm, the old path definitely never existed.

Reply:
that is really really strange - I can see it working as we speak.
@mareklibra @rawagner any idea?

Reply:
@jelkosz maybe openshift had something like an alias in place?

Reply:
Hi @jelkosz  can you please take a look using the browser network debug, find the request line and sharing with us the URL and the response message?

Reply:
@jelkosz maybe it has to do with the console itself. This here: https://github.com/openshift/console/blob/b15c43d6d79997cbaea20ccf7bf694788a882373/README.md#openshift-console, talks about some proxying with this prefix.

Reply:
`/api/kubernetes` is just a proxy (with added auth info) created by `/openshift/console` server. Default kubernetes api server starts at `/`, i.e. `/apis`. We need the prefix, since we are calling console origin and using its authentification : `/api/kubernetes/apis/subresources.kubevirt.io/v1alpha3/healthz`.

Reply:
Oh Thanks @suomiy good to know! 

So the problem for the status is in the web-ui? so who is returning the 503 error?

Reply:
not sure where is the problem -it works on my machine. Can you debug it more thoroughly?